.github/workflows/release.yaml

```diff
@@ -76,3 +86,4 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           HOMEBREW_TAP_GITHUB_TOKEN: ${{ steps.octo-sts.outputs.token }}
+          TAG: ${{ steps.check.outputs.existing_tag || steps.create_tag.outputs.new_tag }}
```

Comment:
will this result in `TAG: ""` if existing_tag  is empty string?

Reply:
I think so, but that can't happen given how this pipeline is set up currently